src/ert/dark_storage/endpoints/ensembles.py

```diff
@@ -42,27 +32,21 @@ def get_ensemble(
     )
 
 
-@router.put("/ensembles/{ensemble_id}/userdata")
-async def replace_ensemble_userdata(
-    *,
-    ensemble_id: UUID,
-    body: Any = DEFAULT_BODY,
-) -> None:
-    raise NotImplementedError
-
-
-@router.patch("/ensembles/{ensemble_id}/userdata")
-async def patch_ensemble_userdata(
-    *,
-    ensemble_id: UUID,
-    body: Any = DEFAULT_BODY,
-) -> None:
-    raise NotImplementedError
-
-
-@router.get("/ensembles/{ensemble_id}/userdata", response_model=Mapping[str, Any])
-async def get_ensemble_userdata(
+@router.get("/ensembles/{ensemble_id}/small", response_model=js.EnsembleOut)
```

Comment:
What do we mean by `small` here?

Reply:
For lack of a better word. Its for getting ensemble data without parameter and response names. Ill refactor this